launch/tier4_perception_launch/launch/traffic_light_recognition/traffic_light.launch.xml

```diff
@@ -171,7 +171,7 @@
     <node pkg="topic_tools" exec="relay" name="fusion_signals_relay" output="screen">
       <param name="input_topic" value="$(var judged/traffic_signals)"/>
       <param name="output_topic" value="$(var output/traffic_signals)"/>
-      <param name="type" value="autoware_auto_perception_msgs/msg/TrafficSignalArray"/>
+      <param name="type" value="autoware_perception_msgs/msg/TrafficSignalArray"/>
```

Comment:
It may be strange that it is TrafficSignal

Reply:
You are right. It should be TrafficLightGroupArray

Reply:
```suggestion
      <param name="type" value="autoware_perception_msgs/msg/TrafficLightGroupArray"/>
```